app/Mailer/MailerServiceProvider.php

```diff
@@ -58,19 +62,31 @@ function () use ($container): MailsSentRepository {
             }
         );
 
+        $logger = LoggerFactory::create($this->container, LoggerName::forResqueWorker('mails'));
+
         $container->addShared(
             SendOwnershipMailCommandHandler::class,
-            function (): SendOwnershipMailCommandHandler {
+            function () use ($logger): SendOwnershipMailCommandHandler {
                 return new SendOwnershipMailCommandHandler(
                     $this->container->get(Mailer::class),
                     $this->container->get(MailsSentRepository::class),
-                    $this->container->get(UserIdentityResolver::class),
                     new TwigEnvironment(
                         new FilesystemLoader(__DIR__ . '/../../src/Mailer/templates'),
                     ),
                     $this->container->get(OwnershipSearchRepository::class),
                     $this->container->get(OwnershipMailParamExtractor::class),
-                    LoggerFactory::create($this->container, LoggerName::forResqueWorker('mails')),
+                    new SendToOwnerOfOwnership(
+                        $this->container->get(UserIdentityResolver::class),
+                        $logger
+                    ),
+                    new CombinedRecipientStrategy(
```

Comment:
Is it not weird to combine a single Strategy?

Reply:
yeah, but in a later pr I will combined it with "ownership_search". 
But now I think of it, I could already write that class, and then just make a PR later to add the projector to fill the table.

Reply:
So I need to wait with the review or you remove the use `CombinedRecipientStrategy` here?

Reply:
I added a new class SendToOwnersOfOrganisation, that fetches all ownerships of an organizer that are approved and sends a mail to them. So feel free to continue the review.